DocSum/ui/gradio/docsum_ui_gradio.py

```diff
@@ -313,11 +313,14 @@ def render(self):
 if __name__ == "__main__":
     import argparse
 
+    import nltk
+
     parser = argparse.ArgumentParser()
     parser.add_argument("--host", type=str, default="0.0.0.0")
     parser.add_argument("--port", type=int, default=5173)
 
     args = parser.parse_args()
     logger.info(">>> Starting server at %s:%d", args.host, args.port)
-
+    nltk.download("punkt_tab")
```

Comment:
I would add a comment in the code to note that these extra downloads are needed for `UnstructuredURLLoader` when reading content from a URL